xla/service/gpu/transforms/dynamic_slice_fusion_rewriter.cc

```diff
@@ -238,15 +248,17 @@ bool IsHandledConstantForDynamicSliceFusion(const HloInstruction& offset) {
 
 // This checks whether a dynamic index operation has all offsets that are either
 // constant or loop iteration offsets.
-bool HasConstantOrLoopIterationOffsets(
-    const HloDynamicIndexInstruction& instr) {
-  return llvm::all_of(instr.index_operands(), [](const HloInstruction* offset) {
-    return IsLoopIterationNumber(*offset) ||
-           IsHandledConstantForDynamicSliceFusion(*offset);
-  });
+bool HasConstantOrLoopIterationOffsets(const HloDynamicIndexInstruction& instr,
+                                       CallGraph* call_graph) {
+  return llvm::all_of(
```

Comment:
Let's use `absl::c_all_of` here too (it is inconsistent to use two different methods to do this in the same file).

Reply:
Done. Thanks.